src/providers/patreon.js

```diff
@@ -0,0 +1,27 @@
+/** @type {import(".").OAuthProvider} */
+export default function Patreon(options) {
+    return {
+        id: 'patreon',
+        name: 'Patreon',
+        type: 'oauth',
+        version: '2.0',
+        authorization: { 
+          url : 'https://www.patreon.com/oauth2/authorize?response_type=code',
```

Comment:
```suggestion
          url : 'https://www.patreon.com/oauth2/authorize',
```

I believe this should be unnecessary